packages/pds/src/db/migrations/index.ts

```diff
@@ -55,3 +55,4 @@ export * as _20230530T213530067Z from './20230530T213530067Z-rebase-indices'
 export * as _20230605T235529700Z from './20230605T235529700Z-outgoing-repo-seq'
 export * as _20230703T044601833Z from './20230703T044601833Z-feed-and-label-indices'
 export * as _20230718T170914772Z from './20230718T170914772Z-sequencer-leader-sequence'
+export * as _20230719T170511044Z from './20230719T170511044Z-invite-note'
```

Comment:
I see this has a conflict with `20230727T172043676Z-user-account-cursor-idx`.  Since that migration has already run, we'll need to rename the migration for `invite-code` so that it comes after `user-account-cursor-idx`.  The simplest way to do this is probably to use the `migration:create` command again to get a new file, and move the contents of the migration into that file.